src/stores/account.ts

```diff
@@ -17,6 +17,10 @@ export const useBcrosAccount = defineStore('bcros/account', () => {
   const userLastName = computed(() => user.value?.lastName || '')
   const userFullName = computed(() => `${userFirstName.value} ${userLastName.value}`)
   const isStaffAccount = computed(() => currentAccount.value.accountType === AccountTypeE.STAFF)
+  const isCAAccount = computed(() => activeProducts?.value?.some(
+    product => product.code === ProductCodeE.CA_SEARCH &&
+    product.subscriptionStatus === ProductStatusE.ACTIVE
+  ))
```

Comment:
@BrandonSharratt , we need to identify what account type this is.

What account type has access to the CA Search product (and the product is Active)?

Is it one of these?

1. regular user
2. REGI staff
3. SBC Staff Tier 2
4. SBC Staff Tier 1
5. MAXIMUS Help Desk
6. SBC Contact Centre (previously called BCOL Help Desk)

Reply:
@cameron-eyds Are you working with Brandon on this?

Reply:
@severinbeauvais Sorry for the delay sir, i am not, no.